tools/bwa.py

```diff
@@ -194,15 +195,7 @@ def align_mem_one_rg(self, inBam, refDb, outBam, rgid=None, options=None,
             # os.unlink(headerFile)
             # os.system("samtools view -h {} > /Users/tomkinsc/Desktop/test_reheader.bam".format(reheadered_bam))
 
-            # sort
-            sorter = tools.picard.SortSamTool()
-            sorter.execute(
-                tmp_bam_aligned,
-                outBam,
-                sort_order='coordinate',
-                picardOptions=['CREATE_INDEX=true', 'VALIDATION_STRINGENCY=SILENT'],
-                JVMmemory=JVMmemory
-            )
+            shutil.move(tmp_bam_aligned, outBam)
```

Comment:
Because `tmp_bam_aligned` was created with `mkstempfname`,  you might need to actually `shutil.copy` instead, or maybe just delete the `else` block and have the `self.mem` call just write directly to `outBam` instead of the temp file.